DOCS-2798: Deploy Istio Ambient Mode on your cluster (#2431)
tigera/docs: calico-enterprise_versioned_docs/version-3.22-2/compliance/istio/deploy-istio-ambient.mdx

---
description: This page explains how to deploy Calico's bundled version of Istio in ambient mode.
---

# Deploy Istio Ambient Mode on your cluster

You can deploy Calico's bundled version of Istio in ambient mode to provide mTLS encryption to your workloads.

## Limitations

* [Application layer network policies](../../network-policy/application-layer-policies/alp.mdx) are not compatible with the Istio service mesh.
* Istio Ambient Mode does not work together with [workload-based web application firewalls](../../threat/web-application-firewall.mdx).
* The service mesh is not supported for use on clusters that are also part of a [cluster mesh](../../multicluster/index.mdx).
* Destination ports are preserved only when Istio is deployed without Waypoint.

>   If you deploy Waypoint, all traffic in the mesh will show port 15008 as its destination port.

Reviewer: ```suggestion
  If you deploy Waypoint, all traffic through waypoints will show port 15008 as its destination port.
```

Author: Done.
